libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -418,10 +418,19 @@ proc validateAndRelay(g: GossipSub,
           break
     toSendPeers.excl(seenPeers)
 
+    #We first send to the outbound peers to avoid peers sending same message to each other
```

Comment:
Could you please elaborate on why that is the case?

Reply:
It was intended to avoid A sending to B and B sending to A at the same time,,, but yes, its not making much impact,,, due to the preceding IDontWants